sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FieldValue.java

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are you adding a public FieldValue to deal with the new receipt changes?

Reply:
I only had the strongly typed `FormField<T>` which could hold any `T` but now we need a `FieldValue` to provide a `type` for conversion for the user.

Similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs#L71).